app/src/main/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenter.java

```diff
@@ -29,13 +29,13 @@
     @Inject
     MainApplication mainApplication;
     @Inject
-    NewsStorage newsStorage;
+    GetFilteredNewsUseCase getUseCase;
```

Comment:
You should keep the naming rules across the app. Imagine that you'll see such a line somewhere:
`setUseCase.get()` 
Would you understand what's going to happen once it gets called?
I don't think so. Your variables names are doing the same - they're not providing any info on what are they going to do:
`getUseCase` - is it going to get a UseCase?
`updateItemsUseCase` - what kind of items is it going to update?
`updateUseCase` - what is it going to update?
Let's pick them a better names.